pkg/kubectl/service.go

```diff
@@ -52,7 +53,8 @@ func paramNames() []GeneratorParam {
 		{"default-name", true},
 		{"name", false},
 		{"selector", true},
-		{"port", true},
+		{"port", false},
+		{"multiport", false},
```

Comment:
Add a comment describing how `port` and `ports` (because of above rename) are different